src/client/src/components/werewolf/NighttimeView.js

```diff
@@ -27,14 +28,20 @@ import {
   ROLE_DOPPELGANGER,
   ROLE_TANNER,
 } from '../../constants';
-import { currPlayerSelector, wakeUpRoleSelector } from '../../store/selectors';
+
+import { currPlayerSelector,
+         currUserIsSpectatorSelector,
+         wakeUpRoleSelector,
+} from '../../store/selectors';
 
 function NighttimeView() {
   const currPlayer = useSelector(currPlayerSelector);
   const wakeUpRole = useSelector(wakeUpRoleSelector);
   const isAwake = currPlayer?.originalRole === wakeUpRole;
+  const isSpectator = useSelector(currUserIsSpectatorSelector);
 
   const renderWakeUp = () => {
+    if (isSpectator) {return (<h3>You should never be awake...</h3>)}
```

Comment:
lol. i think we should just show nothing here. "You should never be awake..." is a little too confusing.

```suggestion
    if (isSpectator) { return false; }
```

Reply:
actually ... maybe we do want to show something. let me think about it.

Reply:
yeah this is just the `renderWakeUp` function so we'd just show this. I think this is acceptable.

![image](https://user-images.githubusercontent.com/665970/97786759-f8d31b00-1b6a-11eb-9e7a-3d051afe3c8e.png)